crates/noirc_evaluator/src/ssa_refactor/ir/basic_block.rs

```diff
@@ -8,10 +12,11 @@ use super::ir::instruction::{Instruction, TerminatorInstruction};
 /// block, then all instructions are executed. ie single-entry single-exit.
 #[derive(Debug, PartialEq, Eq, Hash, Clone)]
 pub(crate) struct BasicBlock {
-    /// Arguments to the basic block.
-    phi_nodes: Vec<BlockArguments>,
+    /// Parameters to the basic block.
+    parameters: Vec<ValueId>,
```

Comment:
`BlockArguments` doesn't exist anymore since `Value::Param` refers to a value that is a block argument.
Additionally, these are ValueIds rather than Values because only the central store in the function's dfg owns all Values.

Reply:
Makes sense